### ISSUE TITLE
Create a Portal Page for EPHS Clubs

### ISSUE THREAD

**Issue by eduxstad** [ephs/ephs.github.io#11]
Now that we have some redirects on ephs.club (/dev, /trace), it would be cool to have a (fancy) list that you could click to get to any EPHS Club. Probably hosted on ephs.club/portal.

**Comment by AlecDusheck** [comment]
I actually had this idea originally when we first got the domain. The plan was to have a list of clubs at EPHS (with a nice material Angular front end) and have links to their websites with descriptions and contact info

**Comment by eduxstad** [comment]
Yeah, sounds good. Do you have any good angular templates ideas for how to present the links?

**Comment by AlecDusheck** [comment]
Possibly... I’ll check when I get back.
